docs/examples/frameworks/jax/jax-multigpu_example.ipynb

```diff
@@ -7,27 +7,58 @@
    "source": [
```

Comment:
<blockquote>that we want to apply to <s>out</s> (our?) computation.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/5185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done</p>